datalad/cmd.py

```diff
@@ -68,6 +68,30 @@
 )
 
 
+class BatchedCommandError(CommandError):
```

Comment:
please add docstring to describe why we need it and what is "request"... with all `requests` and/or weekend took me a bit to understand what "request" it is dealing with,